drivers/zfs/zfs.go

```diff
@@ -300,6 +303,14 @@ func (d *Driver) create(id, parent string, storageOpt map[string]string) error {
 	}
 	if parent == "" {
 		mountoptions := map[string]string{"mountpoint": "legacy"}
+		for _, opt := range strings.Split(d.options.mountOptions, ",") {
+			val := strings.SplitN(opt, "=", 2)
+			if len(val) > 1 {
+				mountoptions[val[0]] = val[1]
+			} else {
+				mountoptions[val[0]] = ""
+			}
+		}
```

Comment:
You're sure this goes here instead of in `Get()` below?

Reply:
Nope, Fixed.